src/Umbraco.Cms.Integrations.DAM.Aprimo/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/Render/AprimoAsset.cshtml

```diff
@@ -31,23 +10,36 @@
         <h3>@Model.Title</h3>
     </div>
 </div>
-@if (Model.CropItemsVM.Any())
+<!-- Crops -->
+@if (Model.MediaWithCrops != null)
 {
     <div class="aprimo-asset-row">
-        <h3>Crop Items</h3>
-        <br/>
-        <div class="aprimo-crops-row">
-            @foreach (var cropItem in Model.CropItemsVM)
+        <h3>Cropped images</h3>
+        @if (Model.MediaWithCrops.Original != null)
+        {
+            <div class="column">
+                <h3>@Model.MediaWithCrops.Original.Name</h3>
+                @if (!string.IsNullOrEmpty(Model.MediaWithCrops.Original.Url))
+                {
+                    <img src="@Model.MediaWithCrops.Original.Url" class="thumbnail" alt="@Model.MediaWithCrops.Original.Name" />
+                }
+            </div>
+        }
+        @if (Model.MediaWithCrops.Crops.Any())
+        {
+            @foreach (var cropItem in Model.MediaWithCrops.Crops)
             {
                 <div class="column">
-                    @if (!string.IsNullOrEmpty(cropItem.PublicLink))
+                    @if (!string.IsNullOrEmpty(cropItem.Url))
                     {
-                        <img src="@cropItem.PublicLink" class="thumbnail" />
+                        <img src="@cropItem.Url" class="thumbnail" alt="@cropItem.Name" />
                     }
-                    <p><b>Label:</b> @cropItem.Label</p>
-                    <p><b>Size:</b> @cropItem.ResizeWidth x @cropItem.ResizeHeight</p>
+                    <h3>@cropItem.Name</h3>
+                    <p>
+                        @cropItem.Extension - @($"{cropItem.ResizeWidth} x {cropItem.ResizeHeight} px")
```

Comment:
Should we use these to add `width` and `height` attributes to the image?

Reply:
The URL points to the resized image, so adding the attributes will not make any difference.
For example this media asset: https://p1.sb.aprimocdn.net/partner1/b648ad4f-743e-4af3-8363-afaf00d4a59d/Wireless%20headset%20listening%20to%20music%20304328931_Social.jpg, which already comes as 1920 x 1080 px.

Reply:
May still be good practice to add them (at least it was in 2005 but I may be out of date now!).  Just as it means the browser reserves space for the image before loading it, so the screen doesn't jump when it has.